### ISSUE TITLE
Crash on AutoFocus Failure

### ISSUE THREAD

**Issue by HashEngineering** [HashEngineering/dash-wallet#30]
```
java.lang.RuntimeException: autoFocus failed
at android.hardware.Camera.native_autoFocus(Native Method)
at android.hardware.Camera.autoFocus(Camera.java:1352)
at de.schildbach.wallet.ui.ScanActivity$AutoFocusRunnable.run(ScanActivity.java:333)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:148)
at android.os.HandlerThread.run(HandlerThread.java:61)
```

**Comment by HashEngineering** [comment]
This may be fixed in the next update.

**Comment by HashEngineering** [comment]
Most of these crashes were resolved in the v5.14 update.
